components/tools/OmeroJava/test/integration/gateway/RawDataFacilityTest.java

```diff
@@ -90,6 +91,26 @@ public void testGetTile() throws DataSourceException, DSOutOfServiceException, D
         Assert.assertEquals(planeData, rawDataPart);
     }
     
+    @Test
+    public void testGetHistogram() throws DataSourceException,
+            DSOutOfServiceException, DSAccessException {
+        ImageData img = browseFacility.getImage(rootCtx, imgId);
+
+        int[] exp = new int[256];
+        for (byte b : rawData) {
+            exp[Byte.toUnsignedInt(b)]++;
```

Comment:
This caused https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-integration/487/ to fail. Is it Java8 specific?